doc/api/crypto.md

```diff
@@ -2800,6 +2800,25 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### `crypto.randomInt(min, max, callback)`
+
+* `min` {integer} Start of random range.
+* `max` {integer} End of random range (non-inclusive).
```

Comment:
Just a thought and nothing that would need to be implemented here, but maybe we should accept bigints as well (and return a bigint in that case)?